tests/lib/rules/no-extra-boolean-cast.js

```diff
@@ -21,6 +21,7 @@ const ruleTester = new RuleTester();
 ruleTester.run("no-extra-boolean-cast", rule, {
 
     valid: [
+        "Boolean(bar, !!baz);",
```

Comment:
Thanks for this fix!

Can we also add a test for the other fix in the default behavior, something like an `invalid` test with `Boolean(Boolean(foo))`.